src/lib/stepper/_stepper-theme.scss

```diff
@@ -20,18 +21,31 @@
     }
 
     .mat-step-icon {
+      color: mat-color($primary, default-contrast);
+    }
+
+    .mat-step-icon-selected {
       background-color: mat-color($primary);
       color: mat-color($primary, default-contrast);
     }
 
-    .mat-step-icon-not-touched {
+    .mat-step-icon-not-selected {
```

Comment:
Rather than having a `selected` and `not-selected` class, why not have the unselected styling as the default and only override the when it's selected?

Reply:
Initially, it was like that but I was having problems overriding the styles.